src/luminol/algorithms/anomaly_detector_algorithms/default_detector.py

```diff
@@ -9,8 +9,8 @@
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 """
-from exp_avg_detector import ExpAvgDetector
-from derivative_detector import DerivativeDetector
+from luminol.algorithms.anomaly_detector_algorithms.exp_avg_detector import ExpAvgDetector
```

Comment:
Imports in Py2 vs Py3 have been an occasionally confusing issue... If we're only supporting Py2.7, I don't _think_ we need the `from __future__ import absolute_import` directive, but may be worth double-checking. Also, I've been told that relative imports, e.g. `from .exp_avg_detector import ExpAvgDetector`, are preferable in subpackage modules but not in `__init__.py` modules, but that may be a matter of preference.

Reply:
This is a good point. Explicit relative imports for subpackage modules seems to be the convention.

Reply:
After trying it out, with the levels we're traversing it would look like:

```
from ... import utils
from ...algorithms.anomaly_detector_algorithms.exp_avg_detector import ExpAvgDetector
from ...algorithms.anomaly_detector_algorithms.derivative_detector import DerivativeDetector
from ...algorithms.anomaly_detector_algorithms import AnomalyDetectorAlgorithm
from ...constants import *
from ...modules.time_series import TimeSeries
```  

I think in this case I'm partial to the absolute imports, but either is fine with me.

Reply:
Yeah, I'm mostly sure it doesn't matter. If this works in both Py2 and Py3, there's probably no real reason to change it.

Reply:
Ok cool, I'll leave it as is for now.

Thank you for the review :)